dbhydra/dbhydra_core.py

```diff
@@ -1682,10 +1677,10 @@ def add_foreign_key(self, foreign_key):
         parent_id = foreign_key['parent_id']
         parent = foreign_key['parent']
         query = "ALTER TABLE " + self.name + " MODIFY " + parent_id + " INT UNSIGNED"
-        print(query)
+        pass
```

Comment:
better full remove than replace for pass - or why is it done like that?

Reply:
It's quite handy to see queries in the console btw - most of the time this is beneficial